cypress/integration/queryBuilder.spec.js

```diff
@@ -44,6 +44,29 @@ describe('test query builder', () => {
     });
   });
 
+  describe('timestamps are displayed as expected', () => {
```

Comment:
Just curious, is there a reason we want / need to test timestamp display in an integration test rather than a unit test?

Reply:
The integration test for the timestamp value is testing to make sure the combination of the output from the API, the formatting by the reducer and the displayed value all amount to the correct value. I do think we should eventually add unit testing on the reducers, but that was a bigger undertaking than I wanted to do right now. I've added a unit test for the handling of null values in the Light table for both singleton and array values.

Reply:
I also think there is value is making sure the timestamp that we're ingesting matches the timestamp we're displaying in the UI since it's going through a couple of layers of conversion.

Reply:
Well-reasoned. No objections here :)